gap/PolygonalComplexes/modification.gi

```diff
@@ -2016,11 +2016,18 @@ InstallMethod( SplitMendableFlagPairs, "for a polygonal complex",
             verts1 := VerticesOfEdges(complex)[pair[1]];
             verts2 := VerticesOfEdges(complex)[pair[2]];
             if Length(Intersection(verts1, verts2)) = 0 then
-                Append( flagPairs, [ 
-                    Set( [ [ verts1[1], pair[1] ], [ verts2[1], pair[2] ] ] ),
-                    Set( [ [ verts1[1], pair[1] ], [ verts2[2], pair[2] ] ] ),
-                    Set( [ [ verts1[2], pair[1] ], [ verts2[1], pair[2] ] ] ),
-                    Set( [ [ verts1[2], pair[1] ], [ verts2[2], pair[2] ] ] )] );
+		if not [verts1[1],verts2[1]] in VerticesOfEdges(complex) and not [verts2[1],verts1[1]] in VerticesOfEdges(complex)  then
```

Comment:
Shouldn't there be a Set around [verts1[1],verts2[1]]? Since VerticesOfEdges(complex) returns Sets?